src/app/modules/verification/verification.model.ts

```diff
@@ -96,17 +107,28 @@ const compileVerificationModel = (db: Mongoose): IVerificationModel => {
   VerificationSchema.statics.createTransactionFromForm = async function (
     form: IFormSchema,
   ): Promise<IVerificationSchema | null> {
-    const { form_fields } = form
-    if (!form_fields) return null
-    const fields = extractTransactionFields(form_fields)
-    if (fields.length === 0) return null
+    const { form_fields, payments_field } = form
+    if (
+      (!form_fields || form_fields.length == 0) &&
+      (payments_field == undefined || !payments_field.enabled)
```

Comment:
/nit

can be shortened
```suggestion
      !form_fields?.length && !payments_field?.enabled
```
or at least made consistent in style
```suggestion
      (!form_fields || form_fields.length === 0) &&
      (!payments_field || !payments_field.enabled)
```